packages/sdks-tests/src/tests/inputs.spec.ts

```diff
@@ -1,7 +1,8 @@
 import { expect } from '@playwright/test';
-import { test } from './helpers.js';
+import { excludeTestFor, test } from './helpers.js';
 
 test.describe('Inputs', () => {
+  test.skip(excludeTestFor({ angular: true }), 'Angular Gen2 SDK not implemented.');
```

Comment:
All of these should be `test.fail()`. 

`test.fail()` will automatically tell you when the test starts to pass, which is useful when we start to implement more features!

`test.skip()` will not run the test at all, so it wont know if the test starts to pass

Reply:
Sure! Let me update these